en/identity-server/7.0.0/docs/apis/organization-apis/authentication.md

```diff
@@ -1,72 +1,102 @@
 
-1. Go to the **Authorized API** tab of [the created application](#register-an-oidc-application).
-2. Authorize the organization APIs you intend to invoke.
+1. On the {{product_name}} Console, go to **Applications**.
+
+2. Select the created application and go to its **API Authorization** tab.
+
+3. Click **Authorize an API Resource** and provide the relevant details of the API resource. Repeat the step for the required APIs
+
+    !!! tip
+        Find the relevant scopes from the API definition of organization APIs.
+
+4. Click **Finish** to save the changes.
 
 !!! note
-    Learn more about [Authorize API to application]({{base_path}}/guides/api-authorization/#authorize-the-api-resources-for-an-app).
+    For more information, refer to [API authorization]({{base_path}}/guides/api-authorization/).
+
+## Step 3: Associate roles with an application
+
+Two requirements must fulfill for a logged in user to receive an access token with the correct permissions.
+
+- The user must be assigned to a role that is associated with the application.
+- The role should permit the user to access the requested API resources and perform the requested actions.
 
-## Assign roles to application
+Applications may be associated with roles made for the organization, or tailored specifically for the application. To associate roles with an application,
 
-1. Go to the **Roles** tab of [the created application](#register-an-oidc-application).
-2. Select Role Audience and associate roles with the necessary permissions for API invocation.
+1. Go to the **Roles** tab of the created application.
+2. Do one of the following:
+    - Select **Role Audience** as **Application** and under **Assigned Roles**, select from the roles created specifically for the application.
+    - Select **Role Audience** as **Organization** to associate all organization-level roles with the application.
+3. Click **Update** to save the changes.
 
 !!! note
-    The user seeking authentication must possess the necessary privileges to invoke APIs within the organization. 
-    This user should be a member of a role that has the required permissions, and this role should be associated with the application in the organization that accesses the API.
+    - Learn more about roles in [manage roles]({{base_path}}/guides/users/manage-roles/).
+    - Learn more about API authorization in [API authorization with RBAC]({{base_path}}/guides/api-authorization/).
 
-## Get the authorization code
-First, your application must initiate a login request to the authorization endpoint of {{ product_name }}. After redirecting to {{ product_name }}, the user should be prompted with a login page if the user is not authenticated.
 
-**Request Format**
-``` bash
-https://{{ host_name }}/oauth2/authorize?response_type=code&redirect_uri={redirect_uri}&client_id={client_id}
-```
+## Step 4: Get the authorization code
```

Comment:
Before this step 

need to mention that, roles associated to the application will be available in the application shared organizations.
At the organization level, the roles need to be assigned to the user

Reply:
Added the steps specifically to complete the flow. Can you review it? @AnuradhaSK 

![screencapture-localhost-8000-asgardeo-docs-apis-organization-apis-authentication-2024-07-08-18_21_15](https://github.com/wso2/docs-is/assets/31583807/686ea8a4-0e27-4445-9473-2a6f9b93de6a)